### PR TITLE
Added amundsen-databuilder

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge/staged-recipes  ready for review

File: recipes/amundsen-databuilder/NOTICE [hunk 1]
```diff
@@ -0,0 +1,4 @@
+amundsencommon
+Copyright 2019-2020 Lyft Inc.
```
Comment (ocefpaf) [inline recipes/amundsen-databuilder/NOTICE:2]: Having this outside of the license text is odd. Makes it hard to interpret if all the code is licensed under Apache-2.0 or not.

Comment (rxm7706) [reply recipes/amundsen-databuilder/NOTICE:2]: @ocefpaf  - True, I wasnt sure if I should include that notice, but i checked setup.py on github and https://www.amundsen.io/ both have the license as Apache-2.0.

Comment (ocefpaf) [reply recipes/amundsen-databuilder/NOTICE:2]: If the license does not require that notice I would not include it. It bring some confusion and make it hard to interpret. Maybe ask upstream about it?

Comment (rxm7706) [reply recipes/amundsen-databuilder/NOTICE]: removed notice

**Pull request merged**